app/scripts/lib/metrics.js

```diff
@@ -62,13 +64,13 @@ define([
   }
 
   function Metrics (options) {
-    /*eslint complexity: [2, 18] */
+    /*eslint complexity: [2, 19] */
```

Comment:
Unless it can be configured to ignore `||`, I think we should. I don't think cyclomatic complexity is a particularly great metric anyway fwiw.